tests/integration/pybaseball/test_statcast_pitcher.py

```diff
@@ -1,11 +1,91 @@
-from pybaseball.statcast_pitcher import statcast_pitcher
+import pandas as pd
+
+from pybaseball.statcast_pitcher import (
+	statcast_pitcher,
+	statcast_pitcher_exitvelo_barrels,
+	statcast_pitcher_expected_stats,
+	statcast_pitcher_pitch_arsenal,
+	statcast_pitcher_arsenal_stats,
+	statcast_pitcher_pitch_movement,
+	statcast_pitcher_active_spin,
+	statcast_pitcher_percentile_ranks
+)
 
 
 def test_statcast_pitcher() -> None:
-    result = statcast_pitcher('2019-01-01', '2019-12-31', 605483)
+    result: pd.DataFrame = statcast_pitcher('2019-01-01', '2019-12-31', 605483)
 
     assert result is not None
     assert not result.empty
 
     assert len(result.columns) == 89
     assert len(result) == 1982
+
+def test_statcast_pitcher_exitvelo_barrels() -> None:
+	minBBE = 100
+	result: pd.DataFrame = statcast_pitcher_exitvelo_barrels(2019, minBBE=minBBE)
+
+	assert result is not None
+	assert not result.empty
+
+	assert len(result.columns) == 19
+	assert len(result) == 432
```

Comment:
We've had issues with asserting exact lengths before when SC just changes the qualifying parameters before, I think we shouldn't have this in the test suite (that was what broke batting)

Reply:
Yeah that's why I specified the minimum number of events for each test, but happy to remove it or change it to just > 0 or something

Reply:
Yeah but we had a break with minBBE too